.editorconfig

```diff
@@ -0,0 +1,15 @@
+; Copyright (c) 2013 Hartmut Kaiser
+;
+; Distributed under the Boost Software License, Version 1.0. (See accompanying
+; file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+; Download plugins for your favorite editor at editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = latin1
```

Comment:
It seems that, where we use non-ASCII text, we are using UTF-8 (as one should), except in LaTeX which is its own world.  Should we be adding a statement to the contrary like this?

Reply:
We can completely remove the `.editorconfig` file now as clang-formatting is in place.

Reply:
I'll remove the file (as a new commit, to preserve your history as I understand is the desire) in the version we merge.